src/Microsoft.Framework.CopyOnWriteDictionary.Internal/Microsoft.Framework.CopyOnWriteDictionary.Internal.kproj

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="__ToolsVersion__" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
VS doesn't do a great job here.  recreate these files using `\\GarnetA\Share\KProjUpdateScript\UpdateKProjectFiles.ps1` before checking in